Remotion/Build/Customizations/Projects.props

```diff
@@ -2,16 +2,51 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <TestExecutionRuntime>net-4.6.1</TestExecutionRuntime>
+    <TestSetupBuildFileForDatabase>$(SolutionDirectory)Build\TestingSetupForDatabase.targets</TestSetupBuildFileForDatabase>
+    <TestSetupBuildFileForWebTest>$(SolutionDirectory)Build\TestingSetupForWebTest.targets</TestSetupBuildFileForWebTest>
   </PropertyGroup>
 
   <PropertyGroup>
-    <TestingSetupBuildFileForDatabase>$(SolutionDirectory)Build\TestingSetupForDatabase.targets</TestingSetupBuildFileForDatabase>
-    <TestingSetupBuildFileForWebTest>$(SolutionDirectory)Build\TestingSetupForWebTest.targets</TestingSetupBuildFileForWebTest>
+    <WebtestingTestConfiguration Condition="'$(WebtestingTestConfiguration)' == ''">
+      Chrome           + NET462 + Debug   + x86 + NoDB + EnforcedLocalMachine;<!-- Upgrade to NET48 once it is supported -->

```

Comment:
remind me, how do we choose the docker image (.net 4.7.2 or .net 4.8) that is used to host IIS?